pkg/controllers/nodeclaim/underutilization/metrics.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package underutilization
```

Comment:
I had some thoughts up above on this, specifically on the semantic differences of finding empty and underutilized nodes. We have them as separate disruption methods right now, since emptiness is a much easier check and doesn't require a scheduling simulation.